src/main/markers/waveform.segments.js

```diff
@@ -25,10 +25,10 @@ define([
       return view;
     });
 
+    var segmentId = 0;
     var createSegmentWaveform = function (options) {
-      var segmentId = "segment" + self.segments.length;
       var segment = {
-        id: segmentId,
+        id: options.id || ('peaks.segment.' + segmentId++),
```

Comment:
I could - we'd need to update waveform-data as well though, which ends up replacing 0`s at https://github.com/bbcrd/waveform-data.js/blob/master/lib/core.js#L194-L195